### PR TITLE
updated the Readme with heroku and expected answers

### PR DESCRIPTION
**Pull request by ConchobarMacNessa**
Hi Shireen – I just updated the readme with my suggestions on the morning challenge re using heroku rather than elephantSQL (relates #13) and I have also added in the expected tables for each of the challenges (relates #14).

Please let me know your thoughts :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,22 +6,21 @@ The purpose of this challenge is to construct complicated queries, learn about j
...
 - cd into the repo in your command line `$ cd db-morning-challenge`
 - Install dependencies with `$ npm install`
-- In your browser, go to [ElephantSQL](https://www.elephantsql.com/)
-- Log into ElephantSQL via GitHub
-- Click on 'Create new instance' to create a new database
-- Give your database a name, choose the 'Tiny Turtly' free plan, and select any data center from the list
-- Click on the name of your new new database to see details; you'll need the URL. Copy this to your clipboard!
+- Create an app on Heroku `heroku create app-name-here --region eu` (If you haven't yet installed Heroku CLI see [here](https://devcenter.heroku.com/articles/heroku-cli)
+- Push to Heroku `git push heroku master`
+- Create a new database on Heroku: `heroku addons:create heroku-postgresql:hobby-dev`
```
Comment (shiryz) [inline README.md]: I would add this as an option to creating the database with an alias `heroku addons:create heroku-postgresql:hobby-dev --as USERS_DB`

File: README.md [hunk 2]
```diff
@@ -6,22 +6,21 @@ The purpose of this challenge is to construct complicated queries, learn about j
...
+- Find the database url on the heroku dashboard for your project, under settings (click reveal config vars)
```
Comment (shiryz) [inline README.md]: you can also use this `heroku config -s | grep USERS_DB` in the command line

File: README.md [hunk 3]
```diff
@@ -31,14 +30,22 @@ One of the tables you have created is a books table that looks like this:
...
 
 *Hint: try using sub queries*
 
+You should expect to see this:
+
+| book_id | book_name                                | max_reservation_time |
+|---------|------------------------------------------|----------------------|
+| 1       | Javascript: The Good Parts               | 21                   |
+| 5       | Pride and Prejudice                      | 21                   |
+
```
Comment (shiryz) [inline README.md]: Awesome :+1:

**Comment by shiryz** [comment]
Just noticed this does link to an issue, can you please add an issue and reference it :smiley_cat:

**Comment by ConchobarMacNessa** [comment]
Thanks @shiryz, just updated per your comments :)

**Pull request merged**